### DISCUSSION TITLE
Add SQL Server conversion support for all data types to the generator code

### DISCUSSION THREAD

**Discussion by tschettler** [dineug/erd-editor#264] [Ideas]
The following contains a conversion table from SQL Server types to equivalent C# types:
https://docs.microsoft.com/en-us/sql/relational-databases/clr-integration-database-objects-types-net-framework/mapping-clr-parameter-data

Many of these conversions are missing from the generator code, such as uniqueidentifier to Guid and various numeric type conversions. Can this listing be reviewed to determine which (if any) conversions can be added to the generator code?

**Comment by dineug** [comment]
Added @vuerd/plugin-generate-template.
Available in vuerd-vscode v0.9.4.
